### DISCUSSION TITLE
Galaxy brain badge

### DISCUSSION THREAD

**Discussion by duyplus** [Schweinepriester/github-profile-achievements#66] [Q&A] [answered]
Can I get badge?

**Comment by ghost** [comment]
You can do it ;)

**Reply by ghost** [reply]
I'll probably get it shortly

**Comment by duyplus99** [comment]
Good luck to you.

**Comment by melvincwng** [comment, marked as answer]
@duyplus, @ps18293, @duyplus99 if you guys are trying to get the Galaxy Brain badge, please refer to #58 for more info.

Hope this helps.

Happy achievement hunting :)

**Reply by duyplus99** [reply]
Wowww, thank you

**Reply by melvincwng** [reply]
@duyplus99, glad that I could help :)

If you would like to stay in touch, kindly follow me on GitHub.

Trying to make more friends/connections on this wonderful platform 😊

**Comment by SamLam140330** [comment]
Yes, you can!!! 
~~Could you please mark me as answer~~
